Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -1122,6 +1122,9 @@ ALWAYS_INLINE ExecutionResult OpCode_JumpNonEmpty::execute(MatchInput const& inp
     }
 
     if (state.string_position < input.view.length()) {
+        auto f = form();
```

Comment:
I'll let @alimpfard review the actual change, but please don't use single-letter variable names here.
```c++
auto form = this->form();
```
would be fine.